fastlane/jetpack_metadata/zh-Hant/subtitle.txt

```diff
@@ -1 +1 @@
-讓你的 WordPress 網站更快更安全。
\ No newline at end of file
+更快速安全的 WordPress 體驗
```

Comment:
19 characters ✅

That is if App Store Connect does something like `"...".length`, which would make sense to me given that the 30 characters limitation is not a matter of sentence complexity but of screen real estate. I'm not actually sure if Chinese characters are counted in the same way as we do with the letters in a Roman language.

Reply:
Can confirm that's how they're counted 🙂 

![image](https://user-images.githubusercontent.com/216089/143448676-ad4e0890-50d4-4e01-983e-2664cf02e228.png)

Reply:
Oh, thank you for checking ❤️ `TIL`